### DISCUSSION TITLE
Sneha Resume

### DISCUSSION THREAD

**Discussion by Snehsk** [recodehive/Stackoverflow-Analysis#349] [Q&A] [answered]
Please review my resume.
[Sneha Resume new web.pdf](https://github.com/user-attachments/files/16324370/Sneha.Resume.new.web.pdf)

**Comment by sanjay-kv** [comment, marked as answer]
[ATS Version of CV (9).docx](https://github.com/user-attachments/files/16342824/ATS.Version.of.CV.9.docx)
[sneha.pdf](https://github.com/user-attachments/files/16342825/sneha.pdf)

**Comment by Snehsk** [comment]
Thanks a lot, I will update my resume according to your guidance 🙂

**Reply by sanjay-kv** [reply]
no worries update and come back to discussion and upload new resume.
